index.js

```diff
@@ -22,16 +20,19 @@ class Action {
         this.nugetSource = process.env.INPUT_NUGET_SOURCE || process.env.NUGET_SOURCE
         this.includeSymbols = JSON.parse(process.env.INPUT_INCLUDE_SYMBOLS || process.env.INCLUDE_SYMBOLS)
         this.throwOnVersionExixts = JSON.parse(process.env.INPUT_THOW_ERROR_IF_VERSION_EXISTS || process.env.THOW_ERROR_IF_VERSION_EXISTS)
+        this.sourceName = "nuget.org"
 
         const existingSources = this._executeCommand("dotnet nuget list source", { encoding: "utf8" }).stdout;
         if(existingSources.includes(this.nugetSource) === false) {
             let addSourceCmd;
             if (this.nugetSource.startsWith(`https://nuget.pkg.github.com/`)) {
                 this.sourceType = "GPR"
-                addSourceCmd = `dotnet nuget add source ${this.nugetSource}/index.json --name=${(SOURCE_NAME)} --username=${this.githubUser} --password=${this.nugetKey} --store-password-in-clear-text`
+                this.sourceName = "nuget.pkg.github.com"
+                addSourceCmd = `dotnet nuget add source ${this.nugetSource}/index.json --name=${(this.sourceName)} --username=${this.githubUser} --password=${this.nugetKey} --store-password-in-clear-text`
             } else {
                 this.sourceType = "NuGet"
-                addSourceCmd = `dotnet nuget add source ${this.nugetSource}/v3/index.json --name=${SOURCE_NAME}`
+                this.sourceName = this.nugetSource
```

Comment:
Hello! Can you elaborate on this, please? 

Why not leaving the previously defined `"nuget.org"` as the source name instead of using its' URL? Is there a reason for this?

Reply:
The reason is that the user can provide their own source via the `NUGET_SOURCE` input parameter, for example `myget.org`. It doesn't make sense to give the name`nuget.org` to what could be any source.